exasol_integration_test_docker_environment/lib/base/ssh_access.py

```diff
@@ -75,8 +75,10 @@ def public_key_as_string(self, comment="") -> str:
         return f"ssh-rsa {b64} {comment}"
 
     def write_public_key(self, path: str, comment="") -> 'SshKey':
+        def opener(path, flags):
+            return os.open(path, flags, 0o600)
         content = self.public_key_as_string(comment)
-        with open(path, "w", mode=0o600) as file:
+        with open(path, "w", opener=opener) as file:
```

Comment:
Oh, see this the first time, but I guess you wanted directly set the correct permissions

Reply:
Initially I did. But on windows (although not an supported use case) Python raised an error.

Reply:
👍